forge-gui/res/cardsfolder/upcoming/glorfindel_dauntless_rescuer.txt

```diff
@@ -0,0 +1,11 @@
+Name:Glorfindel, Dauntless Rescuer
+ManaCost:2 G
+Types:Legendary Creature Elf Noble
+PT:3/2
+T:Mode$ Scry | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigCharm | TriggerDescription$ Whenever you scry, choose one and CARDNAME gets +1/+1 until end of turn.
```

Comment:
no ABILITY?

Reply:
It's pretty hard to get the correct oracle to show up :(

Reply:
you should check ingame

Reply:
I think it's fine 
![image](https://github.com/Card-Forge/forge/assets/67333662/43427106-3b18-4a3e-a105-6835b89d7140)

Reply:
How old is your engine...?

Looks like this on master:
![image](https://github.com/Card-Forge/forge/assets/8506892/53dfc4fc-d84b-418c-9a9a-83cdf2d079f9)